contracts/Compliance.sol

```diff
@@ -1,12 +1,9 @@
 pragma solidity 0.5.16;
 
-import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "./spec_interfaces/ICompliance.sol";
-import "./interfaces/IElections.sol";
+import "./ContractRegistryAccessor.sol";
 
-contract Compliance is ICompliance, Ownable { // TODO consider renaming to something like 'ValidatorIdentification' or make more generic
-
-    IContractRegistry contractRegistry; // TODO move this (and logic) to a super class
+contract Compliance is ICompliance, ContractRegistryAccessor { // TODO consider renaming to something like 'ValidatorIdentification' or make more generic
```

Comment:
comment is obsolete

Reply:
Out of scope for this PR but yes. Will clean this up when converting compliance repr to boolean.